force-app/main/default/flexipages/__utam__/productExplorerPage.utam.json

```diff
@@ -0,0 +1,80 @@
+{
+    "root": true,
+    "selector": {
+        "css": "body"
+    },
+    "elements": [
+        {
+            "name": "navigationBar",
+            "type": "utam-global/pageObjects/appNav",
+            "public": true,
+            "selector": {
+                "css": "one-appnav"
+            }
+        },
+        {
+            "name": "activeFlexiPage",
+            "selector": {
+                "css": ".oneContent.active app_flexipage-lwc-app-flexipage"
+            },
+            "type": "utam-sfdx/pageObjects/threeRegionsTemplate",
+            "public": true
```

Comment:
It seems unnecessary for this element to be public, if all you're going to do is expose the three public elements of the PO returned here under different methods of this PO. Either (1) make this element not public, or (2) make this element public and remove the methods accessing the elements of the referenced PO, and let those exposed methods be the way you get to the inner components.

Reply:
Good catch, I removed the public attribute.